polling_stations/apps/file_uploads/models.py

```diff
@@ -56,7 +56,29 @@ def status(self):
         for f in self.file_set.all():
             if not f.csv_valid:
                 return "Error"
-        return "OK"
+            else:
+                return self.ems_check(f)
+
+    def ems_check(self, file):
+        wait_time = now() - timedelta(seconds=30)
```

Comment:
I think 30 seconds is too short. Lambda can take 10-15 seconds, and ems files can be 5-15mb. I think we can't rely that much on council upload speeds.